### DISCUSSION TITLE
When I clicked on “Tunnel Terminations”, I encountered an error:

### DISCUSSION THREAD

**Discussion by RenJiangZhou2163** [netbox-community/netbox#16201] [Help Wanted!]
When I clicked on “Tunnel Terminations”, I encountered an error:

<class 'django.core.exceptions.FieldError'>

Field 'termination' does not generate an automatic reverse relation and therefore cannot be used for reverse querying. If it is a GenericForeignKey, consider adding a GenericRelation.

Python version: 3.11.4
NetBox version: 3.7.5
Plugins: 
  netbox_interface_sync: 0.2.0
  netbox_topology_views: 3.9.1

**Comment by RenJiangZhou2163** [comment]
same question on netbox version 4.0.2

**Reply by morten-starvik** [reply]
We experience the same on 4.0.8 as well:
<class 'django.core.exceptions.FieldError'>

Field 'termination' does not generate an automatic reverse relation and therefore cannot be used for reverse querying. If it is a GenericForeignKey, consider adding a GenericRelation.

Python version: 3.10.12
NetBox version: 4.0.8
Plugins: 
  netbox_qrcode: 0.0.13
  netbox_topology_views: 4.0.0
